### PR TITLE
feat: 마이페이지 구현

### LINKED ISSUE #82: 마이페이지 구현
## 어떤 기능을 구현하나요?

마이페이지를 구현할 예정입니다.
레시피 작성 먼저 하려고 했는데, 리뷰 작성이 먼저 머지되어야 작업할 수 있을 듯 하여 마이페이지 먼저 작업하겠습니다.

## 일정

- 추정 시간 2시간

### PR DESCRIPTION
**Pull request by hae-on**
## Issue

- close #82 

## ✨ 구현한 기능

- StarRating 구현
- 프로필 기본 이미지 수정
- 마이페이지 수정
- 기존의 member reivew / member recipe page가 나누어져 있던 것을 post page로 합침

스크린샷 2024-04-19 오후 6 55 23

스크린샷 2024-04-19 오후 6 55 05

저장한 꿀조합 같은 경우 새로 api를 파야해서 작업하지 않았습니다.
여기서 하면 코드 너무 많아질 거 같고 회의가 필요한 거 같아서...!

member review와 member recipe의 api가 수정되었습니다.
노션에 수정해두었습니다!
근데 member review는 왜 노션에 안 보이나요...?
원래 따로 없었나요? 걍 제가 추가해놓음....

## 📢 논의하고 싶은 내용
- 오류 발생 (해결 완)

webpack에 있는 VanillaExtractPlugin에 hash 값을 추가해주니까 해결 됨

~아래 사진과 같은 오류가 발생하는데 왜 발생하는지 아시는 분 있나요?(해결 완)~
정확히 MemberRecipeList에서 DefaultRecipeItem 컴포넌트를 추가하면 뜹니다.

스크린샷 2024-04-19 오후 6 53 30

<details>
<summary>오류 복사 내용</summary>
<div markdown="1">
chunk vendors-node_modules_vanilla-extract_webpack-plugin_extracted_js-node_modules_vanilla-extract-0671d1 [mini-css-extract-plugin]
Conflicting order. Following module has been added:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Product/ProductDetailItem/productDetailItem.css.ts.vanilla.css","source":"LnByb2R1Y3REZXRhaWxJdGVtX2NvbnRhaW5lcl9fY2hidmV5MCB7CiAgZGlzcGxheTogZmxleDsKICBmbGV4LWRpcmVjdGlvbjogY29sdW1uOwogIHJvdy1nYXA6IDMwcHg7Cn0KLnByb2R1Y3REZXRhaWxJdGVtX3Byb2R1Y3RJbWFnZV9fY2hidmV5MSB7CiAgd2lkdGg6IDEwMCU7CiAgb2JqZWN0LWZpdDogY292ZXI7Cn0KLnByb2R1Y3REZXRhaWxJdGVtX3Byb2R1Y3RPdmVydmlld19fY2hidmV5MiB7CiAgbWFyZ2luOiAyMHB4IDAgMjVweDsKICBwYWRkaW5nOiAwIDIwcHg7Cn0KLnByb2R1Y3REZXRhaWxJdGVtX3Byb2R1Y3RJbmZvX19jaGJ2ZXkzIHsKICBkaXNwbGF5OiBmbGV4OwogIGp1c3RpZnktY29udGVudDogc3BhY2UtYmV0d2VlbjsKICBhbGlnbi1pdGVtczogZmxleC1zdGFydDsKfQoucHJvZHVjdERldGFpbEl0ZW1fcHJvZHVjdERldGFpbHNfX2NoYnZleTQgewogIGRpc3BsYXk6IGZsZXg7CiAgZmxleC1kaXJlY3Rpb246IGNvbHVtbjsKICBnYXA6IDRweDsKfQoucHJvZHVjdERldGFpbEl0ZW1fY2F0ZWdvcnlOYW1lX19jaGJ2ZXk1IHsKICBjb2xvcjogdmFyKC0tY29sb3JzLWdyYXk0X193dW51c3g3KTsKfQoucHJvZHVjdERldGFpbEl0ZW1fcHJvZHVjdE5hbWVfX2NoYnZleTYgewogIGZvbnQtc2l6ZTogMS42cmVtOwogIGZvbnQtd2VpZ2h0OiA2MDA7CiAgbGluZS1oZWlnaHQ6IDEuNDsKfQoucHJvZHVjdERldGFpbEl0ZW1fc3VtbWFyeVdyYXBwZXJfX2NoYnZleTcgewogIGRpc3BsYXk6IGZsZXg7CiAgZ2FwOiAxMnB4Owp9Ci5wcm9kdWN0RGV0YWlsSXRlbV9wcmV2aWV3V3JhcHBlcl9fY2hidmV5OCB7CiAgZGlzcGxheTogZmxleDsKICBhbGlnbi1pdGVtczogZmxleC1zdGFydDsKICBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsKICBnYXA6IDRweDsKfQoucHJvZHVjdERldGFpbEl0ZW1fcHJvZHVjdENvbnRlbnRfX2NoYnZleTkgewogIG1hcmdpbjogMTZweCAwIDhweDsKICB3aGl0ZS1zcGFjZTogcHJlLWxpbmU7Cn0="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
despite it was not able to fulfill desired ordering with these modules:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Recipe/RecipeProductButton/recipeProductButton.css.ts.vanilla.css","source":"LnJlY2lwZVByb2R1Y3RCdXR0b25fY29udGFpbmVyX18xcXZ5YTh0MCB7CiAgd2lkdGg6IDI2cHg7CiAgaGVpZ2h0OiAyNnB4OwogIGJhY2tncm91bmQ6ICMwMDAwMDA7CiAgYm9yZGVyLXJhZGl1czogNTAlOwp9Ci5yZWNpcGVQcm9kdWN0QnV0dG9uX3RyYW5zbHVjZW50X18xcXZ5YTh0MSB7CiAgb3BhY2l0eTogNTAlOwp9Ci5yZWNpcGVQcm9kdWN0QnV0dG9uX3JlY2lwZVByb2R1Y3RXcmFwcGVyX18xcXZ5YTh0MiB7CiAgbWFyZ2luOiA0OHB4IDIwcHg7Cn0="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Recipe/RecipeFavoriteButton/recipeFavoriteButton.css.ts.vanilla.css","source":"LnJlY2lwZUZhdm9yaXRlQnV0dG9uX2NvbnRhaW5lcl9fbDh3cGtpMCB7CiAgZGlzcGxheTogZmxleDsKICBhbGlnbi1pdGVtczogY2VudGVyOwp9"}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Recipe/RecipeItem/recipeItem.css.ts.vanilla.css","source":"#H4sIAAAAAAAAE51UTY/TQAy991f4UqkrMUu6/WCZ3tgTEgduHKtJ4jSGyXiYcdoC2v+O8lXabLcgLtHYid979nPmPmBGHj8KVluqzA6/BOM9hu12vv5u629ZAr8mAJ4jCbHTENAaoT1uJgCWHKoSaVeKhmQzeZ7cn8H1xwb0hDZv0Q6US6lhniTTBmZAMLVwE1fk1PDJeuGPTS7lkGNQweRURw3rLsvpV8xEFSQaMt5jaJIm+iYZjBBrWMJbWI2lFWbPgQQ/1CLsxi0/jFo2aWRbS9uysNfw2LGHTnYbXeL7wHmdyXX4xQ34lEW4OjFYLG4SPFHI7AvLlv9AME/OGYYop+it+aGhsHib8xNFGfOubqzKK5Yve+LKhB051U9UDXpGrq+64i6r4cEfIbKlHPYmzJTK2HKIqi+yDdZ2e6hdHY/uris8qkg/ye2adXGCTlTKrzV6ubnr880dZI/buKb3EltKCvnnawTv/pcAoCArzUDSdn4OY5ytkund9R+yJ49PXDs5sT/+bed7qm5f+kCCcbHgUOnuaI3gDNQqmb5pn9BOvbVFX3p0KElwMGfxQilaSz5SPMl73w2nqVLRmww1OD4E49tbYI+hsHzQUFKeo2ul4VHUnxcD4Gby/BvVPXVs8gQAAA=="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Members/MemberReviewItem/memberReviewItem.css.ts.vanilla.css","source":"Lm1lbWJlclJldmlld0l0ZW1fdGl0bGVXcmFwcGVyX18xOXB3MnQyMCB7CiAgZGlzcGxheTogZmxleDsKICBqdXN0aWZ5LWNvbnRlbnQ6IHNwYWNlLWJldHdlZW47Cn0KLm1lbWJlclJldmlld0l0ZW1fcmV2aWV3SW1hZ2VfXzE5cHcydDIxIHsKICB3aWR0aDogMTY0cHg7CiAgaGVpZ2h0OiA5MHB4OwogIGJvcmRlci1yYWRpdXM6IDZweDsKICBvYmplY3QtZml0OiBjb3ZlcjsKfQ=="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Members/MemberImage/memberImage.css.ts.vanilla.css","source":"Lm1lbWJlckltYWdlX2NvbnRhaW5lcl9faWttb3E0MCB7CiAgYm9yZGVyLXJhZGl1czogNTAlOwogIG9iamVjdC1maXQ6IGNvdmVyOwp9"}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Members/MembersInfo/memberInfo.css.ts.vanilla.css","source":"Lm1lbWJlckluZm9fbG9nb3V0QnV0dG9uX18xMGdyMHF4MSB7CiAgZmxvYXQ6IHJpZ2h0Owp9Ci5tZW1iZXJJbmZvX3dyYXBwZXJfXzEwZ3IwcXgyIHsKICBkaXNwbGF5OiBmbGV4OwogIGFsaWduLWl0ZW1zOiBjZW50ZXI7Cn0KLm1lbWJlckluZm9fbW9kaWZ5QnV0dG9uX18xMGdyMHF4MyB7CiAgZGlzcGxheTogZmxleDsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGp1c3RpZnktY29udGVudDogY2VudGVyOwogIHdpZHRoOiAxN3B4OwogIGhlaWdodDogMTdweDsKICBtYXJnaW4tbGVmdDogNXB4OwogIGJhY2tncm91bmQ6IHZhcigtLWNvbG9ycy1pY29uLWRlZmF1bHRfX3d1bnVzeG8pOwogIGJvcmRlci1yYWRpdXM6IDUwJTsKfQ=="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/RebuyCheckbox/rebuyCheckbox.css.ts.vanilla.css","source":"LnJlYnV5Q2hlY2tib3hfY29udGFpbmVyX18xZnpudjdjMCB7CiAgZGlzcGxheTogaW5saW5lLWZsZXg7CiAgYWxpZ24taXRlbXM6IGNlbnRlcjsKICBnYXA6IDhweDsKICBjdXJzb3I6IHBvaW50ZXI7Cn0KLnJlYnV5Q2hlY2tib3hfY2hlY2tib3hfXzFmem52N2MxIHsKICBkaXNwbGF5OiBub25lOwp9Ci5yZWJ1eUNoZWNrYm94X2NoZWNrQmFzZV9fMWZ6bnY3YzIgewogIGRpc3BsYXk6IGZsZXg7CiAganVzdGlmeS1jb250ZW50OiBjZW50ZXI7CiAgYWxpZ24taXRlbXM6IGNlbnRlcjsKICB3aWR0aDogMTdweDsKICBoZWlnaHQ6IDE3cHg7CiAgYm9yZGVyLXJhZGl1czogNTAlOwp9Ci5yZWJ1eUNoZWNrYm94X2NoZWNrX2RlZmF1bHRfXzFmem52N2MzIHsKICBiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1jb2xvcnMtaWNvbi1saWdodF9fd3VudXN4cyk7Cn0KLnJlYnV5Q2hlY2tib3hfY2hlY2tfY2hlY2tlZF9fMWZ6bnY3YzQgewogIGJhY2tncm91bmQtY29sb3I6IHZhcigtLWNvbG9ycy1ibGFja19fd3VudXN4OSk7Cn0="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/ReviewRegisterForm/StarRate/starRate.css.ts.vanilla.css","source":"LnN0YXJSYXRlX3N0YXJXcmFwcGVyX19zeWNqcDQwIHsKICBkaXNwbGF5OiBmbGV4OwogIGdhcDogOHB4Owp9Ci5zdGFyUmF0ZV9zdGFySWNvbl9fc3ljanA0MSB7CiAgdHJhbnNpdGlvbjogYWxsIDAuM3MgZWFzZS1vdXQ7Cn0="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/ReviewRegisterForm/ReviewTextarea/reviewTextarea.css.ts.vanilla.css","source":"#H4sIAAAAAAAAE5WST2/bMAzF7/kUugxYDg7stM029bZzexuwo8FIrE1UkQSK/pMN++6DlXhdExdFj3wC3+OP1IaxJxx+4CjACLUJXoA8cl0fD0CdLdXvlVIxJBIKXitGB0I93q/+rDYXzRfl2aHKDgNZabWqyvLT/WQI1pJvtKq2cVTVLo6T+hS8FIl+oVbVZst4mMQ9mOeGQ+dtYYILrFUP/Lk4FanYB7bIhaOmlboeOt+l0a9zY37RygePL3XBYKlLWp0zGU+BPbKQAfcBMK2jA4NtcBY5Uy4CZHHAaUCt7spy0pZIGobjzYywW39kEPI9OLJ5hhm7iqNKYVJfpSBz4DnFLKYkAenST4YYXz7C9uIjwD4F18l5syLhoNX263mnJ9bTWa/9TceMXh7QN9LO/jfZf/HCDszzPPG39fJKr1My6BniOyScg25zkKUUHRy1enKYZwZHjS9I8JC0MugFeZIbiFrdLnP8n1CnNgxzxF2OCBEMyVGr6t3mlqxFP7fvXre/TfeIKUHzj+zLmyu8PvpfY546QfkDAAA="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/ReviewRegisterForm/reviewRegisterForm.css.ts.vanilla.css","source":"LnJldmlld1JlZ2lzdGVyRm9ybV9pdGVtVGl0bGVfX2twYzRyNzAgewogIGZvbnQtc2l6ZTogMS4zcmVtOwogIGZvbnQtd2VpZ2h0OiA2MDA7CiAgbWFyZ2luLWJvdHRvbTogOHB4Owp9Ci5yZXZpZXdSZWdpc3RlckZvcm1fcmVxdWlyZWRNYXJrX19rcGM0cjcxIHsKICBjb2xvcjogdmFyKC0tY29sb3JzLWVycm9yX193dW51c3hjKTsKfQoucmV2aWV3UmVnaXN0ZXJGb3JtX3RhZ0FkZEJ1dHRvbl9fa3BjNHI3MiB7CiAgZGlzcGxheTogZmxleDsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGp1c3RpZnktY29udGVudDogY2VudGVyOwogIHdpZHRoOiAxMDAlOwogIGhlaWdodDogNDRweDsKICBmb250LXNpemU6IDEuNHJlbTsKICBmb250LXdlaWdodDogNTAwOwogIGJvcmRlci1yYWRpdXM6IDZweDsKICBib3JkZXI6IDFweCBzb2xpZCB2YXIoLS1jb2xvcnMtYm9yZGVyLWRlZmF1bHRfX3d1bnVzeGwpOwogIGNvbG9yOiB2YXIoLS1jb2xvcnMtZ3JheTNfX3d1bnVzeDYpOwp9Ci5yZXZpZXdSZWdpc3RlckZvcm1fdGFnTGlzdF9fa3BjNHI3MyB7CiAgZGlzcGxheTogZmxleDsKICBmbGV4LXdyYXA6IHdyYXA7CiAgZ2FwOiA0cHg7CiAgbWFyZ2luLXRvcDogMTJweDsKfQoucmV2aWV3UmVnaXN0ZXJGb3JtX3RhZ0J1dHRvbl9fa3BjNHI3NCB7CiAgZGlzcGxheTogZmxleDsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGdhcDogNHB4OwogIGhlaWdodDogMjhweDsKICBwYWRkaW5nOiAwIDZweDsKICBmb250LXNpemU6IDEuM3JlbTsKICBmb250LXdlaWdodDogNTAwOwogIGJvcmRlci1yYWRpdXM6IDRweDsKICBiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1jb2xvcnMtZ3JheTJfX3d1bnVzeDUpOwp9"}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/ReviewTagList/reviewTagList.css.ts.vanilla.css","source":"LnJldmlld1RhZ0xpc3RfY29udGFpbmVyX18xbnQxcTc2MCB7CiAgZGlzcGxheTogZmxleDsKICBmbGV4LWRpcmVjdGlvbjogY29sdW1uOwogIGdhcDogMzJweDsKfQoucmV2aWV3VGFnTGlzdF9pdGVtVGl0bGVfXzFudDFxNzYxIHsKICBmb250LXNpemU6IDEuNnJlbTsKICBmb250LXdlaWdodDogNjAwOwogIG1hcmdpbi1ib3R0b206IDEycHg7Cn0KLnJldmlld1RhZ0xpc3RfdGFnTGlzdF9fMW50MXE3NjIgewogIGRpc3BsYXk6IGZsZXg7CiAgZmxleC13cmFwOiB3cmFwOwogIGdhcDogNHB4Owp9Ci5yZXZpZXdUYWdMaXN0X2NoZWNrYm94X18xbnQxcTc2MyB7CiAgZGlzcGxheTogbm9uZTsKfQoucmV2aWV3VGFnTGlzdF90YWdMYWJlbF9fMW50MXE3NjQgewogIGRpc3BsYXk6IGlubGluZS1ibG9jazsKICBwYWRkaW5nOiA2cHggMTJweDsKICBib3JkZXItcmFkaXVzOiAzMHB4OwogIGJhY2tncm91bmQtY29sb3I6IHZhcigtLWNvbG9ycy1iYWNrZ3JvdW5kLWNhdGVnb3J5X193dW51c3h1KTsKICBjdXJzb3I6IHBvaW50ZXI7CiAgdXNlci1zZWxlY3Q6IG5vbmU7Cn0KLnJldmlld1RhZ0xpc3RfY2hlY2tib3hfXzFudDFxNzYzOmNoZWNrZWQgKyAucmV2aWV3VGFnTGlzdF90YWdMYWJlbF9fMW50MXE3NjQgewogIGJhY2tncm91bmQtY29sb3I6IHZhcigtLWNvbG9ycy1ncmF5NV9fd3VudXN4OCk7CiAgY29sb3I6IHZhcigtLWNvbG9ycy13aGl0ZV9fd3VudXN4Myk7Cn0="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/ReviewList/reviewList.css.ts.vanilla.css","source":"LnJldmlld0xpc3RfY29udGFpbmVyX19pMmgxZWIwIHsKICBkaXNwbGF5OiBmbGV4OwogIGZsZXgtZGlyZWN0aW9uOiBjb2x1bW47CiAgcm93LWdhcDogNDBweDsKICBwYWRkaW5nOiAwIDIwcHg7Cn0="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/ReviewFavoriteButton/reviewFavoriteButton.css.ts.vanilla.css","source":"LnJldmlld0Zhdm9yaXRlQnV0dG9uX2Zhdm9yaXRlQnV0dG9uX18xMnFscDg4MCB7CiAgZGlzcGxheTogZmxleDsKICBhbGlnbi1pdGVtczogY2VudGVyOwogIGdhcDogNHB4OwogIGN1cnNvcjogcG9pbnRlcjsKfQ=="}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/@vanilla-extract/webpack-plugin/virtualFileLoader/dist/vanilla-extract-webpack-plugin-virtualFileLoader.cjs.js?{"fileName":"src/components/Review/ReviewItem/reviewItem.css.ts.vanilla.css","source":"LnJldmlld0l0ZW1fbWVtYmVySW5mb19fMWM0ZDNiNjAgewogIGRpc3BsYXk6IGZsZXg7CiAgYWxpZ24taXRlbXM6IGNlbnRlcjsKICBnYXA6IDhweDsKfQoucmV2aWV3SXRlbV9tZW1iZXJJbWFnZV9fMWM0ZDNiNjEgewogIGJvcmRlci1yYWRpdXM6IDUwJTsKICBvYmplY3QtZml0OiBjb3ZlcjsKfQoucmV2aWV3SXRlbV9mYXZvcml0ZVdyYXBwZXJfXzFjNGQzYjYyIHsKICBtYXJnaW4tbGVmdDogYXV0bzsKfQoucmV2aWV3SXRlbV9yZXZpZXdJbWFnZV9fMWM0ZDNiNjMgewogIGJvcmRlci1yYWRpdXM6IDZweDsKICBvYmplY3QtZml0OiBjb3ZlcjsKfQoucmV2aWV3SXRlbV9yZXZpZXdDb250ZW50X18xYzRkM2I2NCB7CiAgd2hpdGUtc3BhY2U6IHByZS13cmFwOwp9"}!./node_modules/@vanilla-extract/webpack-plugin/extracted.js
   - couldn't fulfill desired order of chunk group(s) ReviewRegisterPage
   - while fulfilling desired order of chunk group(s) ProductDetailPage
</div>
</details>

- 작성한 꿀조합/리뷰 개수

이거 지금 하드코딩으로 넣어놨는데, 이 값 context로 저장해서 가져오는게 맞을까요??

## 🎸 기타

MemberPostPage의 경우 css가 없는데 폴더에 있습니다.
이거 css 작업하다가 필요 없어져서 css만 삭제한 상태인데,
나중에 스타일 필요해서 그냥 폴더 상태로 두겠습니다!

## ⏰ 일정

- 추정 시간 : 2시간
- 걸린 시간 : 4시간

### CODE REVIEW

File: src/components/Common/SectionHeader/SectionHeader.tsx [hunk 1]
```diff
@@ -13,10 +13,11 @@ interface SectionHeaderProps {
 const SectionHeader = ({ name, link, state }: SectionHeaderProps) => {
   return (
     <div className={container}>
+      {/* Heading 컴포넌트로 교체 weight bold */}
       <h1 className={title}>{name}</h1>
       {link && (
         <Link to={link} state={state}>
-          <SvgIcon variant="arrowRight" width={20} height={20} />
+          <SvgIcon variant="arrowRight" width={20} height={20} style={{ transform: 'translateY(2px)' }} />
```
Comment (Leejin-Yang) [inline src/components/Common/SectionHeader/SectionHeader.tsx:20]: 오 👍

File: src/components/Members/MemberReviewItem/MemberReviewItem.tsx [hunk 2]
```diff
@@ -44,79 +45,36 @@ const MemberReviewItem = ({ review, isPreview }: MemberReviewItemProps) => {
...
+
+      {image && <img className={reviewImage} src={image} alt="작성한 리뷰" />}
+      <div style={{ height: '8px' }} />
+
+      <Text size="caption2" color="sub">
+        {content}
+      </Text>
+      <div style={{ height: '8px' }} />
+
+      <TagList tags={tags} />
```
Comment (Leejin-Yang) [inline src/components/Members/MemberReviewItem/MemberReviewItem.tsx:75]: 요 부분은 상품 리뷰랑 같아서 한번에 묶는거 어떤가요?

File: src/constants/index.ts [hunk 3]
```diff
@@ -59,6 +59,11 @@ export const CATEGORY_TYPE = {
   STORE: 'store',
 } as const;
 
+export const POST_TYPE = {
```
Comment (Leejin-Yang) [inline src/constants/index.ts:62]: 상수화 좋습니다 👍

File: webpack.common.js [hunk 4]
```diff
@@ -48,5 +48,9 @@ module.exports = {
       },
     ],
   },
-  plugins: [new Dotenv(), new VanillaExtractPlugin(), new MiniCssExtractPlugin()],
+  plugins: [
+    new Dotenv(),
+    new VanillaExtractPlugin({ identifiers: ({ hash }) => `prefix_${hash}` }),
```
Comment (Leejin-Yang) [inline webpack.common.js:53]: 버그 픽스 좋습니다

**Review by Leejin-Yang — COMMENTED** [review]
수고했어요 해온 ~
이 페이지가 찐막인가요?
코멘트 확인해주세요

File: src/components/Members/MembersInfo/memberInfo.css.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { vars } from '@/styles/theme.css';
+import { style } from '@vanilla-extract/css';
+
+export const container = style({});
```
Comment (xodms0309) [inline src/components/Members/MembersInfo/memberInfo.css.ts:4]: 얘는 삭제해도 될듯?!

**Review by xodms0309 — APPROVED** [review]
굿굿 고생했어요 해온!!!!

File: src/components/Members/MemberReviewItem/MemberReviewItem.tsx (hunk 2 above)
Comment (hae-on) [reply src/components/Members/MemberReviewItem/MemberReviewItem.tsx]: ReviewItemInfo라는 컴포넌트로 따로 빼고 교체하였습니다!

File: src/components/Members/MembersInfo/memberInfo.css.ts (hunk 5 above)
Comment (hae-on) [reply src/components/Members/MembersInfo/memberInfo.css.ts]: 아니 이게 뭐야 삭제 완.

**Comment by hae-on** [comment]
@xodms0309 @Leejin-Yang 

스크린샷 2024-04-26 오후 7 19 22

프로필 수정 페이지 디자인 수정하였습니다.
맨 위 PR보면 나타나는 오류 webpack에 hash 값 추가해서 사라진 줄 알았는데
또 나타나서 그냥 MiniCssExtractPlugin에 순서 무시하도록 설정해놓았습니다. (무슨 일이 생겨날지는 나도 모름...)
이거 사용 안하고 오류 없애려면, 파일마다 css 파일을 두는게 아니라 하나의 파일에 모아두고 또 다른 파일에서 따로 export를 하더라구요?
그거까지하면 너무 일이 커질 거 같아서 걍 순서 무시 저거 설정해놨슴다

그리고 searchInput 보니까 input에 outline none 해두고, 감싸고 있는 div에는 outline 없더라구요.
outline 없애면 접근성에 안 좋다는 거 본 거 같아서 제가 손댄 memberModifyInput에는 처리해뒀는데,
searchInput의 outline은 무슨 색으로 해야하는지 몰라서 그냥 주석만 달아놨어요....!

암튼 코드 확인 부탁드립니다~~

**Review by xodms0309 — APPROVED** [review]
확인완 ~ 고생했어요 해온! 그 outline 디자인엔 없다고 해서 없앴는데 outline 다 통일하는거 어떤가요

**Comment by hae-on** [comment]
넵! 그럼 전체 outline 다 노란색으로 통일할게요~ 작업은 다른 이슈에서 하겠슴니당~~!

**Review by Leejin-Yang — APPROVED** [review]
마이페이지도 고고~

**Pull request merged**